### DISCUSSION TITLE
when " , " is in title and misses from other site

### DISCUSSION THREAD

**Discussion by magnaux** [Purfview/Discogs-Scout#20] [General]
https://www.discogs.com/release/134826-Nine-L-Houston-We-Have-A-Problem
![εικόνα](https://user-images.githubusercontent.com/107857394/216313913-50b1fa90-cc30-48aa-8813-59aaac6cf666.png)

also i trying to understand why this https://www.discogs.com/master/2397286-Nine-L-Selected-Works-93-94
shows yellow allthough it exists and has same name

**Comment by magnaux** [comment]
ohh... it searches Selected Works '93 '94 .... without " - "

**Comment by Purfview** [comment]
Looks like dupe of https://github.com/Purfview/Discogs-Scout/issues/3
Post issues in Issues.
